Impossible-Agent-Suite/DangerousActivities/README.md

```diff
@@ -0,0 +1,19 @@
+# Impossible Finance Dangerous Activities
+
+## Description
+
+This agent detects transactions involving impossible addresses and dangerous addresses
+
+## Supported Chains
+
+- BSC
+
+## Alerts
+
+- impossible-dangerous
+  - Fired when a transaction involves impossible addresses and dangerous addresses
```

Comment:
Throughout the README I'd recommend capitalizing any mention of Impossible Finance.
```suggestion
  - Fired when a transaction involves Impossible Finance addresses and dangerous addresses
```